test/test_operations.py

```diff
@@ -500,6 +501,35 @@ def test_empty_advanced_indexing(self):
     xla_result = xla_base[:, torch.empty(0, 6, dtype=torch.int64)]
     self.assertEqual(result, xla_result)
 
+  def test_grad_gradInput(self):
```

Comment:
What is this for, related to `empty_strided()`?

Reply:
This is a repro example in #911 , where triggers a segfault, which turns out to be caused by missing override for `empty_strided`.

Reply:
The name is a bit misleading.
Even though the `empty_strided()` is well hidden within the test, I think it is better to name the tests for the purpose they have been created.
Like `test_empty_strided()`?